content/guides/core/automations/_index.md

```diff
@@ -6,4 +6,36 @@ menu:
 title: Automations
 weight: 4
 ---
+{{% pageinfo color="info" %}}
+{{< readfile file="/_includes/enterprise-only.md" >}}
+{{% /pageinfo %}}
 
+This page describes _automations_ in W&B. [Create an automation]({{< relref "create-automations/" >}}) to trigger workflow steps, such as automated model testing and deployment, based on an event in W&B, such as when an [artifact]({{< relref "/guides/core/artifacts" >}}) artifact version is created ({{< relref "/guides/core/registry/" >}}).
+
+For example, an automation can post to a Slack channel when a new version is created, or can run a webhook to trigger automated testing when the `production` alias is added to an artifact.
+
+## Overview
+An automation can run when a specific [event]({{< relref "automation-events.md" >}}) occurs in a registry or project.
+
+For an artifact in a [Registry]({{< relref "/guides/core/registry/">}}), you can configure an automation to run:
+- When a new artifact version is linked to a collection. For example, trigger testing and validation workflows for new candidate models.
+- When an alias is added to an artifact version. For example, trigger a deployment workflow when the `deploy` alias is added to a model version.
+
+For an artifact in a [project]({{< relref "/guides/models/track/project-page.md" >}}), you can configure an automation to run:
+- When a new version is added to an artifact. For example, start a training job when a new version of a dataset artifact is added to a given collection.
+- When an alias is added to an artifact version. For example, trigger a PII redaction workflow when the alias "redaction" is added to a dataset artifact.
+
+For more details, refer to [Automation events and scopes]({{< relref "automation-events.md" >}}).
+
+To [create an automation]({{< relref "create-automations/" >}}), you:
+
+1. If required, configure [secrets]({{< relref "/guides/core/secrets.md" >}}) for sensitive strings the automation requires, such as access tokens, passwords, or sensitive configuration details. Secrets are defined in your **Team Settings**. Secrets are most commonly used in webhook automations to securely pass credentials or tokens to the webhook's external service without exposing it in plain text or hard-coding it in the webhook's payload.
+1. Configure the webhook or Slack notification to authorize W&B to post to Slack or run the webhook on your behalf. A single automation action (webhook or Slack notification) can be used by multiple automations. These actions are defined in your **Team Settings**.
+1. In the project or registry, create the automation:
+  1. Define the [event]({{< relref "#automation-events" >}}) to watch for, such as when a new artifact version is added. 
```

Comment:
These nested numbers are not rendering correctly. They appear as a continuation of the list.

Reply:
Done (needed two more spaces each)